lib/transactional.pm

```diff
@@ -163,6 +166,13 @@ sub trup_call {
         }
     }
 
+    # If we expect a reboot on success, the trup-0- might not reach the console.
+    # Check for t-u's own output instead.
+    if ($cmd =~ /reboot / && $args{exit_code} == 0) {
+        wait_serial(qr/transactional-update finished/, timeout => $args{timeout}) || die "transactional-update didn't finish";
```

Comment:
This will not work, there is no string `transactional-update finished` after reboot.
https://openqa.suse.de/tests/6103265#step/rebootmgr/10

Reply:
It's supposed to be there before the reboot. That it doesn't appear means that transactional-update didn't even finish before the reboot happens.

CC @laenion

Reply:
This seems to be a bug in `transactional-update`. I changed it to match a string just before the reboot is triggered as a workaround.